rats-tls/src/attesters/tdx-ecdsa/cleanup.c

```diff
@@ -6,9 +6,9 @@
 
 #include <rats-tls/log.h>
 #include <rats-tls/attester.h>
-#include "../../verifiers/tdx/tdx.h"
+#include "../../verifiers/tdx-ecdsa/tdx.h"
```

Comment:
Do we need to change "tdx.h" to "tdx_ecdsa.h" at the same time for unified style?